react/components/EXPERIMENTAL_useCheckboxTree/index.tsx

```diff
@@ -27,24 +28,28 @@ export default function useCheckboxTree<T>({
 
   const toggle = useCallback(
     (item: T | Tree<T>): void => {
-      if (!isDisabled(item))
+      if (!isDisabled(item)) {
         dispatch({
           type: ActionType.Toggle,
           itemToToggle: { item, nodesKey, comparator },
           isDisabled,
         })
+        setLastToggled(item)
+      }
     },
     [checkedItems]
   )
 
+  useEffect(() => {
+    if (onToggle) {
+      onToggle({ checkedItems, disabledItems, item: lastToggled })
```

Comment:
could be `onToggle && onToggle({ checkedItems, disabledItems, item: lastToggled })`?

Reply:
yes, it could